docs/consuming_with_cmake.rst

```diff
@@ -0,0 +1,91 @@
+.. _consuming_with_cmake:
+
+Consuming with CMake
+==================== 
+
+The best way to use oneMKL is through integration with CMake. This can be done in two ways:
+
+* By using ``find_package`` with an installed version of oneMKL interface library.
+* By downloading and building oneMKL as part of your application using FetchContent_.
+
+
+.. _using_from_installed_binary:
+
+Using an installed oneMKL interface library
+#######################################
+
+If oneMKL has been previously installed, either by building from source or as a distributed
```

Comment:
```suggestion
If the oneMKL interfaces has been previously installed, either by building from source or as a distributed
```

Reply:
Done in [931ee0f](https://github.com/oneapi-src/oneMKL/pull/510/commits/931ee0f68ff1bd61a5f9f7c1957559a4a41a2215).